site/gatsby-node.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path')
+
+module.exports.onCreateNode = ({ node, actions, getNode }) => {
+  if (node.internal.type === 'MarkdownRemark') {
+    const fileNode = getNode(node.parent)
+    const parsedFilePath = path.parse(fileNode.relativePath)
+    const { sourceInstanceName } = fileNode
+
+    if (sourceInstanceName === 'protocols') {
```

Comment:
Can we get rid of nested If?

Reply:
yes as for now, but in future there might be `sourceInstanceName == pages`  for example